src/pages/write-smart-contracts/clarinet.md

```diff
@@ -184,8 +184,8 @@ Clarinet.test({
 });
 ```
 
-For more information on assertions, review [asserts][] in the Deno standard library. For more information on the available
-Clarity calls in Deno, review the [Deno Clarity library][].
+For more information on assertions, review [asserts][] in the Deno standard library. For more information on the
```

Comment:
looks like the URLs are missing, right?

Reply:
These are [reference-style Markdown links](https://daringfireball.net/projects/markdown/syntax#link). I think they make the raw Markdown easier to read. You can see where they are defined at the bottom of the file.

Reply:
wow, **TIL**